packages/agents/sdk/src/interfaces/index.ts

```diff
@@ -29,3 +31,20 @@ export type AssetData = {
   key: string;
   id: string;
 };
+
+export const NxtpSdkXCallArgsSchema = Type.Object({
```

Comment:
Change to SdkXCallArgsSchema cause I believe we are going to purge all "nxtp" references eventually.

Reply:
Agreed, no new code should ever use NXTP.